src/components/CohortFacilitatorList.tsx

```
@@ -38,6 +38,7 @@ const CohortLearnerList: React.FC<CohortLearnerListProp> = ({
   const isMobile = useMediaQuery(theme.breakpoints.down('md'));
   const [loading, setLoading] = React.useState<boolean>(false);
   const [userData, setUserData] = React.useState<UserDataProps[]>();
+  const [resData, setResData] = React.useState<UserDataProps[]>();
```

ESLint

[error] 'setResData' is assigned a value but never used.

(no-unused-vars)